src/contrib/testkits/Akka.TestKit.Xunit2/Akka.TestKit.Xunit2.csproj

```diff
@@ -4,14 +4,15 @@
   <PropertyGroup>
     <AssemblyTitle>Akka.TestKit.Xunit2</AssemblyTitle>
     <Description>TestKit for writing tests for Akka.NET using xUnit.</Description>
-    <TargetFrameworks>$(NetStandardLibVersion)</TargetFrameworks>
+    <TargetFramework>$(NetStandardLibVersion)</TargetFramework>
     <PackageTags>$(AkkaPackageTags);testkit;xunit</PackageTags>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
   </PropertyGroup>
 
   <ItemGroup>
     <ProjectReference Include="..\..\..\core\Akka.TestKit\Akka.TestKit.csproj" />
     <PackageReference Include="xunit" Version="$(XunitVersion)" />
+    <PackageReference Include="FluentAssertions" Version="$(FluentAssertionsVersion)" />
```

Comment:
Do we need the dependency on FluentAssertions for the `TaskExtensions` class?

Reply:
I guess not, I wanted to introduce the `ShouldCompleteWithin` method in tests; I guess we can use our own `WithTimeout` method.

Reply:
I'm fine bringing it in since we seem to use it everywhere, but maybe we should make it part of the base testkit if we're going to take a dependency on it. We can punt on it for now and just do our own thing, or we can try to close out https://github.com/akkadotnet/akka.net/issues/5432

Reply:
How would you like to proceed on that?

Reply:
well, we can't because FluentAssertion depends on XUnit, that's why I'm putting it here instead in the base TestKit

Reply:
Oops, didn't know that FluentAssertion is unit test framework agnostic... I'm recaftoring this to TestKit instead.